tools/release-ci/pkg/config/batch.go

```diff
@@ -41,14 +44,15 @@ type Repo struct {
 // Batch contains configuration for a batch job
 type Batch struct {
 	Job
+	RepoMeta
 	Repo
 }
 
 // GcsPath is the base path for job data. Batch jobs
 // store their information only in the pull request
 // directory, not for any specific PR
 func (b *Batch) GcsPath() string {
-	return fmt.Sprintf("%s/%s/%d/", PullRequestDirectory, b.JobName, b.BuildNumber)
+	return fmt.Sprintf("%s/%s/%s/%s/%d/", PullRequestDirectory, b.RepoOwner, b.RepoName, b.JobName, b.BuildNumber)
```

Comment:
Convention is `org_repo` not `org/repo` -- I don't like it but not sure churn to make it cleaner is worth it

Reply:
Will fix